elasticsearch/pom.xml

```diff
@@ -54,6 +54,31 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Where do you get this one from? It's not in Central, so one needs to add oss.sonatype.org to the local settings.xml?